.github/workflows/maven_central_sync.sh

```diff
@@ -9,20 +9,24 @@ if [ "${GITHUB_TAG:-}" != "" ]; then
   BINTRAY_SUBJECT=${BINTRAY_SUBJECT:-hedgehogqa}
   BINTRAY_REPO=${BINTRAY_REPO:-scala-hedgehog-maven}
 
-  BINTRAY_PACKAGES="hedgehog-core hedgehog-runner hedgehog-sbt hedgehog-minitest"
+  BINTRAY_PACKAGES="hedgehog-core hedgehog-runner hedgehog-sbt"
 
   echo "PROJECT_VERSION: $PROJECT_VERSION"
   echo "BINTRAY_SUBJECT: $BINTRAY_SUBJECT"
   echo "   BINTRAY_REPO: $BINTRAY_REPO"
 
   for bintray_package in $BINTRAY_PACKAGES
   do
+    echo ""
     echo "bintray_package: $bintray_package"
     echo "Sync to Maven Central..."
     curl \
       --user "${BINTRAY_USER}:${BINTRAY_PASS}" \
       -X POST \
+      -H "Content-Type: application/json" \
       "https://api.bintray.com/maven_central_sync/$BINTRAY_SUBJECT/$BINTRAY_REPO/$bintray_package/versions/$PROJECT_VERSION"
+
+    sleep 10s
```

Comment:
Why the sleeP?

Reply:
@charleso First of all, the problem caused by parallel sync with Travis was gone, but now we have a different issue. 
One sync succeeds and the second fails and the same for the third and fourth. 

The error message says 
> "Failed to close repository: Failed to close repository: qahedgehog-1022

So with `sleep`, I'm trying to give it some time to close it properly so that it can process next one.
I'm not sure if 10 seconds are enough though.

***
Charles, there's another issue I got
> [error] java.lang.RuntimeException: error uploading to qa.hedgehog/hedgehog-sbt_2.12/cd3814e793ed055efc03918ae588fa30f2248283/jars/hedgehog-sbt_2.12.jar: You have exceeded your API call limit

It's about `sbt publish`. So we probably need to reconsider releasing every `master` build. 

> ### Limits
> API queries are limited according to the following rules: Limits only apply to resources that do not belong to a user or to the organization the user is part of. The current limit per non-Premium user is 300 queries a day, and 1440 per month. Limits are returned in the X-RateLimit headers. For example:

from https://www.jfrog.com/confluence/display/BT/Bintray+REST+API#BintrayRESTAPI-AccessandVersioning

This "`Limits only apply to resources that do not belong to a user or to the organization the user is part of`" part doesn't make sense, does it? I thought the key set for Hedgehog repo belonged to the organization.